ports/faiththeunholytrinity/faiththeunholytrinity/tools/patchscript

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+LOGFILE="$GAMEDIR/patchlog.txt"
+LOGERR="$GAMEDIR/patch_error.txt"
+
+# Redirect output and error to the log file
+rm -rf $LOGFILE $LOGERR
+exec > >(tee -a "$LOGFILE") 2>&1
+echo "GAMEDIR is set to: $GAMEDIR"
+
+# Exports
+export DATADIR="$GAMEDIR/assets"
+export DATAFILE="$GAME_NAME.port"
+export SDL_GAMECONTROLLERCONFIG="$sdl_controllerconfig"
+export TOOLDIR="$GAMEDIR/tools"
+export TMPDIR="$GAMEDIR/tmp"
+export PATH="$GAMEDIR/tools:$PATH"
+
+# Permissions
+chmod 666 /dev/uinput
+chmod 777 "$TOOLDIR/gmKtool.py"
+chmod 777 "$TOOLDIR/oggenc"
+
```

Comment:
I've missed that too, as you are using resampling `oggdec` is also needed and needs execution flag (although in most cases it will work when installing from PM GUI)

```suggestion
chmod 777 "$TOOLDIR/oggdec"

```